hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java

```diff
@@ -156,6 +162,38 @@ final Stream<DynamicTest> tokenAirdropToExistingAccountsTransfers() {
                                         .hasTokenBalance(FUNGIBLE_TOKEN, 10));
             }
 
+            @HapiTest
+            final Stream<DynamicTest> tokenMultipleAirdropsToSameAccount() {
+                String receiver = "OneReceiver";
+                return defaultHapiSpec("to multiple accounts should transfer fungible tokens")
```

Comment:
Isn't there a way to use the new format for Hapi tests here?
Applies for the rest as well.